libs/SalesforceSDK/build.gradle

```diff
@@ -2,6 +2,8 @@
 apply plugin: 'android-library'
 
 dependencies {
+  compile 'com.android.support:appcompat-v7:23.0.0'
+  compile 'com.facebook.react:react-native:0.11.+'
```

Comment:
React native is on maven central.
We don't actually need the support library here (but I think we will need it in applications using react native). Will remove.